editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -191,25 +170,132 @@ export function runGridRearrangeMove(
         updateGridControlsCommand,
       ]
     }
-    case 'reorder': {
-      return [
-        reorderElement('always', pathForCommands, absolute(possiblyReorderIndex)),
-        deleteProperties('always', pathForCommands, [
-          PP.create('style', 'gridColumn'),
-          PP.create('style', 'gridRow'),
-          PP.create('style', 'gridColumnStart'),
-          PP.create('style', 'gridColumnEnd'),
-          PP.create('style', 'gridRowStart'),
-          PP.create('style', 'gridRowEnd'),
-        ]),
-        updateGridControlsCommand,
-      ]
-    }
     default:
       assertNever(moveType)
   }
 }
 
+export function runGridMoveReorder(
+  targetElement: ElementPath,
+  selectedElement: ElementPath,
+  jsxMetadata: ElementInstanceMetadataMap,
+  interactionData: DragInteractionData,
+  gridCellGlobalFrames: GridCellGlobalFrames,
+  gridTemplate: GridContainerProperties,
+  newPathAfterReparent?: ElementPath,
+): CanvasCommand[] {
+  const common = getGridMoveCommonData(
```

Comment:
it is just stylistic, but I don't really like the variable name `common` and `getGridMoveCommonData`. I know it is hard, because it is just a bunch of info necessary for both rearrange and reorder...
Maybe `calculateGridMoveConfig`?